deny.toml

```
@@ -20,7 +20,7 @@ allow = [
 ]
 
 exceptions = [
-    { name = "webpki-roots", allow = ["MPL-2.0"], version = "*" },
+    { name = "webpki-roots", allow = ["MPL-2.0", "CDLA-Permissive-2.0"], version = "*" },
```

```suggestion
    { name = "webpki-roots", allow = ["CDLA-Permissive-2.0"], version = "*" },
```
I think the MPL-2.0 was replaced by CDLA-Permissive-2.0.

---

Good catch changed it and also updated to have my commit signed forgot the -S flag